### DISCUSSION TITLE
Wiki docker restarting loop

### DISCUSSION THREAD

**Discussion by bjerrecs** [requarks/wiki#4004] [Error / Bug Report] [answered]
Was working a couple of days ago, now not.
Can see it has something to do with Let's Encrypt Certs renewal dry-run?

Currently running on the Digitalocean prebuild image.

![image](https://user-images.githubusercontent.com/32055984/119249539-89ba2400-bb99-11eb-88ea-2c7b31ae87d2.png)

**Comment by skledarg** [comment]
It looks like Wiki.js thinks you changed the wiki domain in between letsencrypt renewals, and starts the renewal process.
When Let's Encrypt requests a validation token from your server at `http://[your.domain.com]/.well-known/acme-challenge/[token]`, Wiki.js should return the challenge token from it's config, but for some reason it looks like your `WIKI.config.letsencrypt.challenge.token` is either not set in your case, or Let's Encrypt gets an empty response from something in front of your wiki.

Without knowing more about your setup and settings, it's hard to say whether there's something wrong with your ssl settings in config, or something else... do you have a reverse proxy or something else (nginx / cloudflare, etc) set up for your wiki that might interfere with the request?

Also check your [Let's Encrypt configuration](https://docs.requarks.io/en/install/config#lets-encrypt) and / or try renewing manually from the admin panel (SSL settings).

You could also try [deleting the wiki container](https://docs.requarks.io/install/upgrade) and pulling / running a new one to make sure you are on the latest version.

**Comment by bjerrecs** [comment, marked as answer]
Sorry for the bad bug report. Was in a rush to get to some family event.

Some days ago I installed the wiki.js wiki on a digital ocean droplet with the rebuild image and followed the setup guide.
All was working fine with SSL and yesterday it broke on me.

Before posting I tried to remove the wiki docker and pull it. Not helping.

After running the step 5 in https://docs.requarks.io/install/digitalocean it now works.
Looks like it removed the challage and made a new
![image](https://user-images.githubusercontent.com/32055984/119276625-6e8ff880-bc1b-11eb-9faa-6f28101b0227.png)
This discussion can be closed :) Thnaks for the pointer! Great software
